Orange/base.py

```diff
@@ -87,6 +88,10 @@ class Learner(ReprableWithPreprocessors):
     #: A sequence of data preprocessors to apply on data prior to
     #: fitting the model
     preprocessors = ()
+    FittedParameter = namedtuple(
```

Comment:
Is there a reason for using `namedtuple` and not `NamedTuple`? (I prefer the latter, also because of type annotation.)